lib/yggdrasil/GuardianErrorHandler.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Yggdrasil.GuardianErrorHandler do
+  @behaviour Guardian.Plug.ErrorHandler
+
+  import Plug.Conn
+
+  def unauthenticated(conn, _params) do
+    conn
+      |> send_resp(:unauthorized, "")
```

Comment:
Plug.Conn.Status defines it as :unauthorized for 401. It resolves the status based on the atom. Unfortunately there is no unauthenticated and seems 401 is what went it covers both unauthenticated and unauthorized from what I have read.